nncf/quantization/algorithms/weight_compression/weight_lowering.py

```diff
@@ -101,7 +101,7 @@ def calculate_normalized_weight_and_nf4_scale(
 
 
 def do_integer_quantization(
-    weight: Tensor, reduction_axes: ReductionAxes, config: WeightCompressionConfig
+    weight: Tensor, reduction_axes: ReductionAxes, config: WeightCompressionConfig, precomputed_scale: Tensor = None
```

Comment:
I would recommend to split function to 2 functions instead of adding new argument.

Reply:
Done